lib/sedna/core/federated_learning/federated_learning.py

```diff
@@ -143,3 +147,30 @@ def train(self, train_data,
                     task_info,
                     K8sResourceKindStatus.RUNNING.value,
                     task_info_res)
+
+class FederatedLearningv2(JobBase):
```

Comment:
I don't think redefine a new class `FederatedLearningv2`,why don't update the old one?

Reply:
After discussion, we keep two interfaces in this PR, will update them at the next PR.